app/src/main/res/layout/activity_continuous_scan.xml

```diff
@@ -0,0 +1,171 @@
+            android:id="@+id/barcode_scanner"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+
+        <ImageView
+            android:id="@+id/toggle_beep"
+            android:layout_width="25dp"
+            android:layout_height="25dp"
+            android:layout_margin="@dimen/spacing_small"
+            app:srcCompat="@drawable/ic_volume_off_white_24dp" />
+
+        <ImageView
+            android:id="@+id/toggle_flash"
+            android:layout_width="25dp"
+            android:layout_height="25dp"
+            android:layout_gravity="end"
+            android:layout_margin="@dimen/spacing_small"
+            app:srcCompat="@drawable/ic_flash_off_white_24dp" />
+
+        <ImageView
+            android:id="@+id/toggle_camera"
+            android:layout_width="25dp"
+            android:layout_height="25dp"
+            android:layout_gravity="end"
+            android:layout_marginEnd="50dp"
+            android:layout_marginRight="50dp"
+            android:layout_marginTop="@dimen/spacing_small"
+            android:scaleX="1.7"
+            android:scaleY="1.7"
+            app:srcCompat="@drawable/ic_switch_camera" />
+
+    </FrameLayout>
+
+    <RelativeLayout
+        android:id="@+id/quick_view"
+        android:layout_width="match_parent"
+        android:layout_height="120dp"
+        android:background="@color/white"
+        android:visibility="gone"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/frame"
+        tools:visibility="visible">
+
+        <ImageView
+            android:id="@+id/quickView_image"
+            android:layout_width="80dp"
+            android:layout_height="match_parent"
+            android:layout_margin="@dimen/activity_horizontal_margin"
+            android:adjustViewBounds="true"
+            android:scaleType="fitCenter"
+            android:src="@drawable/img_scan" />
+
+        <ProgressBar
+            android:id="@+id/quickView_imageProgress"
+            android:layout_width="80dp"
+            android:layout_height="match_parent"
+            android:layout_margin="@dimen/activity_horizontal_margin"
+            android:padding="@dimen/activity_horizontal_margin"
+            android:visibility="gone" />
+
+        <TextView
+            android:id="@+id/quickView_name"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/activity_horizontal_margin"
+            android:layout_marginRight="@dimen/activity_horizontal_margin"
+            android:layout_marginTop="@dimen/activity_horizontal_margin"
+            android:layout_toEndOf="@id/quickView_image"
+            android:layout_toRightOf="@id/quickView_image"
+            android:singleLine="true"
+            android:textSize="18sp"
+            android:textStyle="bold"
+            tools:text="Huzaifa Iftikhar" />
```

Comment:
?

Reply:
It's okay to use `tools:text` just for preview usage, it won't go into the final build ;)

Reply:
Oops I forgot to erase it. @jinqian True it won't go into the final build, still I will remove it, doesn't look good ;)